webpack.config.js

```diff
@@ -45,7 +48,15 @@ const webConfig = dev.getDefaultWebpackConfig({
         '../build/default/bufferutil': 'commonjs ../build/default/bufferutil',
     },
     target: 'webworker',
-
+    resolve: {
```

Comment:
I don't think you need this line. `getDefaultWebpackConfig` doesn't actually extract out the options for `resolve.alias`. And if it's working without it, then I guess that's okay.